docs/src/main/sphinx/sql/drop-schema.md

```diff
@@ -33,6 +33,12 @@ Drop the schema `archive`, along with everything it contains:
 DROP SCHEMA archive CASCADE
 ```
 
+Drop the schema `archive`, only if there are no dependent objects:
```

Comment:
What is a dependent object of a schema. I think you mean to say ... only if there are no objects contained in the schema.